src/cockpit/389-console/src/dsModals.jsx

```diff
@@ -142,11 +144,9 @@ export class CreateInstanceModal extends React.Component {
         }
 
         if (this.state.createDBCheckbox) {
-            for (const attr of optionalAttrs) {
-                if (this.state[attr] === "") {
-                    all_good = false;
-                    errObj[attr] = true;
-                }
+            if (!valid_db_name(this.state.createDBName) || this.state.createDBName === "") {
```

Comment:
Probably, `this.state.createDBName === ""` can be removed as valid_db_name checks for an empty string.